pkg/iac/adapters/terraform/google/storage/adapt.go

```diff
@@ -121,7 +121,7 @@ func (a *adapter) adaptBucketResource(resourceBlock *terraform.Block) storage.Bu
 
 	if logBlock := resourceBlock.GetBlock("logging"); logBlock.IsNotNil() {
 		bucket.Logging.Metadata = logBlock.GetMetadata()
-		bucket.Logging.LogBucket = logBlock.GetAttribute("target_bucket").AsStringValueOrDefault("", logBlock)
+		bucket.Logging.LogBucket = logBlock.GetAttribute("log_bucket").AsStringValueOrDefault("", logBlock)
```

Comment:
Should we rethink the `.AsStringValueOrDefault()` approach? The reason I say it is because if it's not set the default will be the empty value `""`. 

Would using a "known" placeholder be better than an empty value like `""`? For instance, that would eliminate the issue where a value is mistakenly forgotten to be set vs. actually being empty due to a bug.

Reply:
@nikpivkin I think we can handle this in a different PR, WDYT?

Reply:
OK - I created https://github.com/aquasecurity/trivy/issues/9341 to track.